bindings/python/iota_sdk/client/_high_level_api.py

```diff
@@ -59,6 +58,11 @@ class GenerateAddressesOptions():
 class HighLevelAPI():
     """High level API.
     """
+    __metaclass__ = ABCMeta
```

Comment:
From where did you see it like this? Looks like since Python 3 this should be defined like this
```suggestion
class HighLevelAPI(metaclass=ABCMeta):
    """High level API.
    """
```

Reply:
hmm the internet told me that :) ill switch it!